ws/client/connection.py

```diff
@@ -160,6 +160,8 @@ def set_argparser(argparser):
         """
         import ws.config
         group = argparser.add_argument_group(title="Connection parameters")
+        group.add_argument("--site", metavar="NAME",
+                help="site name used in file naming schemes")
```

Comment:
This is not a connection parameter, so it should stay in `ws.config` next to the other global parameters.